lib/iris/aux_factory.py

```diff
@@ -566,9 +449,7 @@ def calc_bounds():
                     orography = orography_pts.reshape(
                         orography_pts_shape.append(1))
                 return self._derive(delta, sigma, orography)
-            b_shape = self._shape(nd_values_by_key)
-            b_dtype = self._dtype(nd_values_by_key)
-            bounds = _LazyArray(b_shape, calc_bounds, b_dtype)
+            bounds = calc_bounds()
```

Comment:
@pp-mo I chose not to because the bounds calculation code is much longer and it was simpler to just leave the inline function and call it.